go/cmd/dolt/commands/tblcmds/mv.go

```diff
@@ -87,51 +85,18 @@ func (cmd MvCmd) Exec(ctx context.Context, commandStr string, args []string, dEn
 		return 1
 	}
 
-	force := apr.Contains(forceParam)
-
-	working, verr := commands.GetWorkingWithVErr(dEnv)
-	if verr != nil {
-		return commands.HandleVErrAndExitCode(verr, usage)
-	}
-
 	oldName := apr.Arg(0)
 	newName := apr.Arg(1)
-
-	if doltdb.HasDoltPrefix(oldName) {
-		return commands.HandleVErrAndExitCode(
-			errhand.BuildDError("error renaming table %s", oldName).AddCause(doltdb.ErrSystemTableCannotBeModified).Build(), usage)
+	queryStr := ""
+	if force := apr.Contains(forceParam); force {
+		queryStr = fmt.Sprintf("DROP TABLE IF EXISTS `%s`;", newName)
 	}
-
-	if verr = schcmds.ValidateTableNameForCreate(newName); verr != nil {
-		return commands.HandleVErrAndExitCode(verr, usage)
-	}
-
-	_, ok, err := working.GetTable(ctx, oldName)
-
-	if err != nil {
-		verr = errhand.BuildDError("error: failed to read tables from working set").Build()
-		return commands.HandleVErrAndExitCode(verr, usage)
-	}
-	if !ok {
-		verr = errhand.BuildDError("Table '%s' not found.", oldName).Build()
-		return commands.HandleVErrAndExitCode(verr, usage)
-	}
-
-	has, err := working.HasTable(ctx, newName)
-
-	if err != nil {
-		verr = errhand.BuildDError("error: failed to read tables from working set").AddCause(err).Build()
-		return commands.HandleVErrAndExitCode(verr, usage)
-	} else if !force && has {
-		verr = errhand.BuildDError("Data already exists in '%s'.  Use -f to overwrite.", newName).Build()
-		return commands.HandleVErrAndExitCode(verr, usage)
-	}
-
-	working, err = working.RenameTable(ctx, oldName, newName)
-	if err != nil {
-		verr = errhand.BuildDError("Unable to remove '%s'", oldName).Build()
-		return commands.HandleVErrAndExitCode(verr, usage)
-	}
-
-	return commands.HandleVErrAndExitCode(commands.UpdateWorkingWithVErr(dEnv, working), usage)
+	queryStr = fmt.Sprintf("%sRENAME TABLE `%s` TO `%s`;", queryStr, oldName, newName)
+
+	cli.CliOut = ioutil.Discard // display nothing on success
```

Comment:
This could (will) take a long time for big tables. Batch mode does print update stats as it goes, leave output alone